Add how to register/resign to the statute (#111)
oemof/oemof: legal/EN_satzung.md

## § 6 Membership

1. Natural persons of full age as well as legal entities can become members of the association.
2. The written declaration of membership is made to the board of directors.
3. The board decides on admission. The admission takes place after free discretion.
    In case of refusal, the board is not obliged to inform the applicant of the reasons for refusal.
4. Membership ends by resignation from the association, by death, exclusion or deletion from the list of members.

>     Resignation takes place at the end of the year,
>     the declaration of resignation has to be filed to the board in written form and at least one month in advance.

Author: ```suggestion
    Resignation has to be filed to the board in written form with a deadline of at least four weeks.
```
